### PR TITLE
Task/14814 14816 adapt changes home screen app info

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Changes to Home Screen and App Info

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14814
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14816

### CODE REVIEW

**Review by 30mar — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift [hunk 1]
```diff
@@ -12,7 +12,8 @@ class AppInformationViewController: DynamicTableViewController, NavigationBarOpa
 	init(
 		elsService: ErrorLogSubmissionProviding,
 		finishedDeltaOnboardings: [String: [String]],
-		cclService: CCLServable
+		cclService: CCLServable,
+		isHibernationState: Bool? = CWAHibernationProvider.shared.isHibernationState
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift:16]: Why optional `Bool`?

File: src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewControllerTests.swift [hunk 2]
```diff
@@ -29,6 +29,21 @@ class AppInformationViewControllerTests: XCTestCase {
 		fakeCCSService = nil
     }
 
+	func testModel_WHEN_modelEntriesInHibernation_THEN_equalCount() throws {
+		// WHEN
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewControllerTests.swift]:
```suggestion
		// GIVEN
```

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/More/HomeMoreInfoTableViewCell.swift [hunk 3]
```diff
@@ -13,20 +13,25 @@ class HomeMoreInfoTableViewCell: UITableViewCell {
...
 		titleLabel.text = AppStrings.Home.MoreInfoCard.title
 
-		for item in MoreInfoItem.allCases {
+		var items: [MoreInfoItem] = []
+		stackView.removeAllArrangedSubviews()
+		
+		if CWAHibernationProvider.shared.isHibernationState {
+			items = MoreInfoItem.allCases.filter({
+				$0 != .settings && $0 != .share
+			})
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/More/HomeMoreInfoTableViewCell.swift]:
```suggestion
			items = MoreInfoItem.allCases.filter { $0 != .settings && $0 != .share }
```

**Review by flxschmidt — APPROVED** [review]
Nice 🍒

File: src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift:16]: So we don't need to change the code at other places (optional + default value)

**Pull request merged**